app.py

```diff
@@ -0,0 +1,19 @@
+from flask import Flask, jsonify
+import json
+import pdb
+
+# instantiate the app
+app = Flask(__name__)
+app.config.from_object(__name__)
+
+@app.route('/emoji', methods=['GET'])
+def get_emoji_list():
+    try:
+        with open('results.json') as json_file:
+            data = json.load(json_file)
+            return jsonify(data)
+    except:
+        return jsonify('error: could not fetch emoji :(')
```

Comment:
Curious, does this return a 500 response still?

Reply:
nope! i figured it's all /* TODO TODO CODE HERE TODO */

Reply:
ok cool